internal/tracinginterceptor/interceptor.go

```diff
@@ -177,12 +177,69 @@ func (i *Interceptor) CallOneway(ctx context.Context, req *transport.Request, ou
 
 // HandleStream implements interceptor.StreamInbound
 func (i *Interceptor) HandleStream(s *transport.ServerStream, h transport.StreamHandler) error {
-	panic("implement me")
+	parentSpanCtx, _ := i.tracer.Extract(i.propagationFormat, getPropagationCarrier(s.Request().Meta.Headers.Items(), i.transport))
+	transportRequest := &transport.Request{
+		Caller:    s.Request().Meta.Caller,
+		Service:   s.Request().Meta.Service,
+		Procedure: s.Request().Meta.Procedure,
+		Headers:   s.Request().Meta.Headers,
+		Transport: s.Request().Meta.Transport,
+	}
+
+	extractOpenTracingSpan := &transport.ExtractOpenTracingSpan{
+		ParentSpanContext: parentSpanCtx,
+		Tracer:            i.tracer,
+		TransportName:     s.Request().Meta.Transport,
+		StartTime:         time.Now(),
+		ExtraTags:         commonTracingTags,
+	}
+	_, span := extractOpenTracingSpan.Do(context.Background(), transportRequest)
```

Comment:
Do we have a reason to use context.Background instead of s.Context()?

Reply:
Thanks! updated to s.Context()